lib/cucumber/cli.js

```diff
@@ -89,6 +89,8 @@ function Cli(argv) {
 \n\
 --no-source                   Don't print the source uris.\n\
 \n\
+--[no-]colors                 Enable/disable colors in output.\n\
```

Comment:
Well okay, if JVM uses this different `--monochrome` flag (I don't think "monochrome" is the right term for this), let's use our preferred `--no-colors`  then.